### ISSUE TITLE
[BUG]: Castlevania Lament of innocence

### ISSUE THREAD

**Issue by Isma939** [PCSX2/pcsx2#11480]
### Describe the Bug

Best regards, you will see in my backup copy, I have always been able to access the game, without problem, but for 2 days, in the save station and with direx 11 or 12 when starting the save, from memory card, everything comes out black, including if I leave the save room, not happening like that in Vulcan, opgl.

I also tried in native resolution since I have it at x 3.

If in direct 11 or 12, I'm talking about two devices, my desktop (I7 4770, 16 ram ddr3, RTX 2070 super) and Rog ally, given that it is a bug... or so I think.

Thank you and I admire your work 

Excuse my English

### Reproduction Steps

When loading saved, you see the entire black environment, in direct 11 or 12 libraries, not in Vulcan or opngl, I also tried native, since I use it x3

### Expected Behavior

I have never had this behavior before, this being the first time.

### PCSX2 Revision

v.1.7.5641

### Operating System

Windows 10 (64bit)

### CPU

I7 4770

### GPU

RTX 2070 Super

### GS Settings

![Captura PNG 1](https://github.com/PCSX2/pcsx2/assets/174081423/1e5fbbe3-7130-4869-acf0-57858ab48af5)
![Captura PNG 2](https://github.com/PCSX2/pcsx2/assets/174081423/7b67a2dc-ee22-49f4-bf47-256fa1a22add)
![Captura PNG 3](https://github.com/PCSX2/pcsx2/assets/174081423/7a80c718-0e2e-4204-a6a1-d50c151cb825)

**Comment by Isma939** [comment]
Thank you, for your response, I am now sending you the screenshots of Asus rog Ally, as you can see the same result, only different hardware, and it did not happen before (approximately 2-3 days) with DX, sorry if I am upset with this, I apologize most sincerely. THANK YOU
![Captura de pantalla 2024-06-28 122636](https://github.com/PCSX2/pcsx2/assets/174081423/e76cf066-6a98-446a-aeb7-360d9fe7a60e)
![Captura de pantalla 2024-06-28 122733](https://github.com/PCSX2/pcsx2/assets/174081423/5b5d2518-adb0-4666-be6e-ca161c6ce3bd)
![Captura de pantalla 2024-06-28 122813](https://github.com/PCSX2/pcsx2/assets/174081423/f31f4aa2-bc83-4dc3-8971-3c945e2d1d06)

**Comment by stenzek** [comment]
Screenshots don't help us. We need a GS dump.

**Comment by Isma939** [comment]
[Castlevania Isma939 GS dump..zip](https://github.com/user-attachments/files/16028615/Castlevania.Isma939.GS.dump.zip)

**Comment by stenzek** [comment]
This never worked correctly in DirectX, in the older builds, the shadow was missing completely.

**Comment by Isma939** [comment]
I always had it, on desktop with DX12, and on Asus Rog Ally DX 11 or 12.

And I have it defaulted to DX12 for other dums (onimusha, GoW 1 and 2) and I have not noticed that they worked poorly.
This is the only case that this has happened to me, he told me and sorry for my lack of knowledge, that for both hadwards (desktop I7 4770, RTX 2070 super and Asus Rog ally) is Vulcan suitable?

I don't understand what the solution could be for DX if neither 11 or 12 works well.
Could you summarize a solution?

Thank you very much, very grateful for your time

**Comment by refractionpcsx2** [comment]
the automatic setting will pick the best renderer for you

**Comment by Isma939** [comment]
Well, it's a shame, a very big shame, since the default is Vulcan, and DX was much smoother for me, FPS and smoothness, I won't ask you if I can somehow make DX 11 or 12 work correctly... it would be a heavy...
Anyway thanks. I can put DX 12 in the GoW and customize castlevania in vulcan or default, anyway thank you very much

**Comment by refractionpcsx2** [comment]
Stens PR will fix DX for you, but Vulkan is recommended for you as it  *should* run well (at least on your desktop) and the blending capability is MUCH higher than direct3d, so the quality of the graphics is much better in a lot of games.
